addon/pods/components/frost-list-item/component.js

```diff
@@ -9,8 +10,12 @@ export default Ember.Component.extend({
     this.set('_frostList', this.nearestOfType(FrostList))
   }),
 
-  isSelected: Ember.computed.reads('model.isSelected'),
-
+  isSelected: Ember.computed('model.isSelected', function () {
+    let modelIsSelect = this.get('model.isSelected')
+    modelIsSelect ? $(this.get('element')).parent().addClass('is-selected')
+    : $(this.get('element')).parent().removeClass('is-selected')
```

Comment:
This could be `$(this.get('element')).parent()[modelIsSelect ? 'addClass' : 'removeClass']('isSelected')`, to remove some extra lines